app_book/views.py

```diff
@@ -3,6 +3,7 @@
 from django.db import transaction
```

Comment:
Es wäre gut, wenn du die Änderungen in `feature/B27-SchaufensterView` machen könntest. Der Branch basiert auf deinem und dann gibt es weniger Konflikte. :-)